pkg/config/config.go

```diff
@@ -1111,6 +1109,11 @@ func (c *RuntimeConfig) ValidateRuntimes() error {
 			logrus.Warnf("'%s is being ignored due to: %q", name, err)
 			failedValidation = append(failedValidation, name)
 		}
+		if c.DefaultRuntime == name {
+			if err := c.TranslateMonitorFields(handler); err != nil {
```

Comment:
It looks like there's no default set for MonitorPath, as defaultRuntimeHandler doesn't set one and TranslateMonitorFields is only called for c.DefaultRuntime. Without runtime_monitor_path explicitly set in the config file, I think this means MonitorPath will be an empty string for any non-default runtime handlers.

Reply:
good point! I've updated to run if the runtime is of type `oci`